docs/docs/server.md

```diff
@@ -1,21 +1,21 @@
-# Zero Server
+# WorkFlow Launcher Server
 
-We now have the basics of Zero
+We now have the basics of WorkFlow Launcher
 running as a server
 in Google App Engine (GAE).
 
 ## Deploy to Google App Engine
 
-To build and deploy Zero,
+To build and deploy WFL,
```

Comment:
Should we spell out Workflow Launcher for all of these replacements in the docs?

Reply:
I thought about it but realized it would be too verbose... I think having the title as `WorkFlow Luncher (WFL)` should be enough to setup the context for the readers of this doc, what do you think?

Reply:
That's fine. I agree it's verbose.